spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -3084,6 +3084,74 @@ describe('Redfish Systems Root', function () {
             });
     });
 
+    it('should throw 500 internal error for post volume with Name error', function() {
+        var redfishVolumeCreate = {
+            "username": "someuser",
+            "password": "somepassword",
+            "volume": {
+                "Id": "AnId",
+                "Name": "name 123",
+                "CapacityBytes": 1234567,
+                "VolumeType": "NonRedundant",
+                "Links": {
+                    "Drives@odata.count": 3,
+                    "Drives": [{
+                        "@odata.id": "/redfish/v1/Systems/SomeNodeId/Storage/RAID.Integrated.1-1/Drives/0"
+                    },
+                    {
+                        "@odata.id": "/redfish/v1/Systems/SomeNodeId/Storage/RAID.Integrated.1-1/Drives/1"
+                    },
+                    {
+                        "@odata.id": "/redfish/v1/Systems/SomeNodeId/Storage/RAID.Integrated.1-1/Drives/2"
+                    }]
+                }
+            }
+        };
+        waterline.catalogs.findLatestCatalogOfSource.resolves(Promise.resolve({
```

Comment:
Promise.resolve() is not necessary, any value returned from promise is a promise already.

Reply:
ok